### ISSUE TITLE
Translate core disability "get benefit" content into Spanish

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#45746]
## Story

As a Veteran with limited English proficiency, I need the core benefit information for VA disability in my primary language.

## What's needed

- [x] **Step 1:** Send a word count and links to pages to State Department for estimate.
- [x] **Step 2:** Send Word docs of 3 pages to be translated:
-  Eligibility
- How to apply
- After you apply
- [x] **Step 3:** Build page in Drupal
- [x] **Step 4:** Send PDF to State Department for QA

**Comment by RLHecht** [comment]
duplicate. closing.

**Comment by RLHecht** [comment]
We sent the first 3 pages to the State Department on 9/23. Attaching here too. [https://app.zenhub.com/files/133843125/e46d96ae-5fae-4108-9676-90513a0e47b0/download](https://app.zenhub.com/files/133843125/e46d96ae-5fae-4108-9676-90513a0e47b0/download)

**Comment by aliyahblackmore** [comment]
Hi @megzehn ! These pages are ready for you and I've included some notes.

We still need the meta descriptions for each of these pages - I am going to pass those along to Randi via email. For the time being I've put in the English as a placeholder as the field is needed in order to save the Drupal page. 

Randi noted that there isn't anything for us to do at the moment with the "Menu settings" in Drupal. Last, I've included the links where needed to the new Spanish pages. I'll flag in standup if we need to think about the order of publishing (i.e. How to file a claim is linked on the Eligibility page).

Elegibilidad para beneficios por discapacidad de VA
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=50389
**Notes:** 

1. On the English page, we previously had a react widget but at the moment it looks like we don't need one. Pinged Randi about deleting it from the live English page and replacing the content with the new text. I've included "¿Cómo presento una reclamación?" in a Rich text field. Also note that in this H2 the note is only on the Spanish page, as it is regarding translation needs.
2. In the featured content box, the bullet point “[Una enfermedad causada por el tiempo que pasó como prisionero de guerra (POW, por sus siglas en inglés) (en inglés)](https://www.va.gov/disability/eligibility/former-pows/)” has the double “en ingles.” Wondering if there is a way to avoid this – as it visually is repeated. Just flagging.
3. In the first H2 of the main content, changed “dos” to “2.”


Cómo presentar una reclamación de discapacidad de VA
Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=50435
**Notes:**

1. I removed “en inglés” from the bullet point “[Averigüe si tiene derecho a recibir compensación por discapacidad de VA (en inglés)](https://www.va.gov/disability/eligibility/)” because I linked to the Spanish version of the page.
2. In the text "Los siguientes documentos le ayudarán a apoyar su reclamación de discapacidad de VA.” The translation has a period but our English page uses a colon. Wondering if there a reason for the period instead of the colon."  I’ve changed to a colon for the time being, but could we flag this with the translation team? cc @RLHecht 
4. I updated the number of days for disability claims to match what was on the English page.  I also changed “agosto” to “septiembre,” for the month.

Proceso de reclamación de VA después de presentar su reclamación
Preview: http://preview-prod.vfs.va.gov/preview?nodeId=50453 
**Notes:** 

1. Updated the number of days to 106.4 and changed “agosto” to “septiembre”
2. In the H2 “El tiempo que toma revisar su reclamación de discapacidad de VA depende de lo siguiente” the bullet points in the Spanish translation include periods. I am leaving these in for the time being but wondering if the translators can provide any grammar context for the choice? Flagging as I know there may be a Spanish grammar rule that applies here. cc: @RLHecht 
3. Similarly, in step 3 of the process list, periods are used in the bulleted list. Could we flag to confirm with the translators that these should be used? cc: @RLHecht

**Comment by megzehn** [comment]
Confirming that these have all been copyedited, but I'm holding in drafts. 

Thanks for updating the month and days for the disability claim number, @aliyahblackmore.

**Comment by RLHecht** [comment]
@aliyahblackmore We got back the meta descriptions for these pages. Could you please put them into Drupal and let me know if there's an issue with the character count? Thanks!
[https://app.zenhub.com/files/133843125/e6048eaa-9391-4e8d-ac8d-e2713c1d1760/download](https://app.zenhub.com/files/133843125/e6048eaa-9391-4e8d-ac8d-e2713c1d1760/download)

**Comment by aliyahblackmore** [comment]
11/10 update: Translators sent back Tagalog meta descriptions so we are waiting to get the Spanish content back.

**Comment by RLHecht** [comment]
@aliyahblackmore Here are the Spanish meta descriptions back from the State Department: [https://app.zenhub.com/files/133843125/1b1c4e06-9600-4d8e-a1cc-d9694c0894e7/download](https://app.zenhub.com/files/133843125/1b1c4e06-9600-4d8e-a1cc-d9694c0894e7/download)

**Comment by aliyahblackmore** [comment]
Thank you, Randi! 

As mentioned, I emailed you the meta description for the "Elegibilidad para beneficios por discapacidad de VA" page.

@megzehn these pages are ready with the meta description and I also updated the average number of days for claims (I just added "102.4" and "octubre"). Depending on when these publish, we'll just need to be sure that we peek of the number of days and update accordingly.

**Cómo presentar una reclamación de discapacidad de VA**
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=50435

**Proceso de reclamación de VA después de presentar su reclamación**
**Preview:** http://preview-prod.vfs.va.gov/preview?nodeId=50453

**Comment by megzehn** [comment]
Thanks, @aliyahblackmore! Confirming that I reviewed these meta descriptions. And I left these in drafts. CC @RLHecht

**Comment by RLHecht** [comment]
I sent the disability eligibility meta description to the translators on 11/22. I'll post here when they return it.

**Comment by RLHecht** [comment]
@aliyahblackmore Here's the meta description for you: [https://app.zenhub.com/files/133843125/8b0564cd-f596-40fd-b3dc-3c09e9f9e625/download](https://app.zenhub.com/files/133843125/8b0564cd-f596-40fd-b3dc-3c09e9f9e625/download)

**Comment by aliyahblackmore** [comment]
Thanks @RLHecht! 

As we discussed, I shortened this meta description a bit so that we can send it back to the translation team. 
I was able to get this down to 253 characters (with spaces). 

We have room for 300 characters total and I think that aiming for about 250/253 will leave some room for the text to grow when it's translated. 

[Eligibility for VA disability benefits shorter meta description_12.14.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/10229359/Eligibility.for.VA.disability.benefits.shorter.meta.description_12.14.docx)


Let me know if you think this would work/if I should have this reviewed by anyone else!

**Comment by RLHecht** [comment]
Thanks @aliyahblackmore. I had @laurwill take a look and she was able to make it a bit more succinct. I'm attaching here and am going to send to the translators so they can get started on it. Thanks again for this!

[https://app.zenhub.com/files/133843125/69b53fa6-8b3d-4ad2-8ee2-839a890c4675/download](https://app.zenhub.com/files/133843125/69b53fa6-8b3d-4ad2-8ee2-839a890c4675/download)

**Comment by DanielleThierryUSDSVA** [comment]
@RLHecht I copied you on the awareness email to our disability SME. So we're good to move forward with publishing translated disability pages and updated English pages. Thanks!

**Comment by RLHecht** [comment]
@aliyahblackmore Here's the last meta description we were waiting on. [https://app.zenhub.com/files/133843125/e0c49f2e-f7c4-4332-a0b7-89a94c545cb3/download](https://app.zenhub.com/files/133843125/e0c49f2e-f7c4-4332-a0b7-89a94c545cb3/download)

**Comment by aliyahblackmore** [comment]
Thanks Randi! 

@megzehn the meta description for the Elegibilidad para beneficios por discapacidad de VA page is ready for you.
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=50389

**Comment by megzehn** [comment]
Thanks, @aliyahblackmore!! @RLHecht, these are published and ready for the QA phase.

**Comment by RLHecht** [comment]
These pages are all live and I've sent the links to the State Department for QA. I'll set up a new ticket for any changes we need to make. Closing this one!
